k8s/base/gcp_gke/deployments.yaml

```diff
@@ -23,6 +23,20 @@ spec:
                 name: mpm-configmap-backend
             - secretRef:
                 name: mpm-secret-backend
+          volumeMounts:
+            - name: storage-data
+              mountPath: /var/www/html/storage
+              subPath: ""
+            - name: logs-data
+              mountPath: /var/www/html/storage/logs
+              subPath: logs
+      volumes:
+        - name: storage-data
+          persistentVolumeClaim:
+            claimName: mpm-storage-pvc
```

Comment:
I have actually never done this. But will this be a problem if a single PVC is shared between pods?

Reply:
From the k8s docs multiple pods can share a single PVC with the right access mode. I used `ReadWriteOnce` which allows pods from "the same" node read and write to the volume. 

https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes